Command.fs

```diff
@@ -135,7 +135,7 @@ module Compose =
     ///     The next available intermediate stage number.
     ///     If the expression has no intermediate stages, we return 0.
     /// </returns>
-    and nextBoolIntermediate =
+    and nextBoolIntermediate : BoolExpr<Sym<_>> -> bigint =
```

Comment:
How exactly does F# interpret the _ in a type?

Is it equivalent to a (forall a. a) type or something?